GitVersion.yml

```diff
@@ -0,0 +1,3 @@
+branches:
+  master:
+    regex: ^main$
```

Comment:
There should be an empty new line in this file.

Reply:
Done.